pkg/config/documentation_test.go

```diff
@@ -0,0 +1,125 @@
+package config_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/lunarway/shuttle/pkg/config"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestShuttleProjectContext_Documentation(t *testing.T) {
+	tt := []struct {
+		name    string
+		planRef string
+		docsRef string
+
+		result string
+		err    error
+	}{
+		{
+			name:    "empty context",
+			planRef: "",
+			docsRef: "",
+			result:  "",
+			err:     errors.New("exit code 1 - Could not find any plan documentation"),
+		},
+		{
+			name:    "unknown plan protocol",
+			planRef: "something-odd",
+			docsRef: "",
+			result:  "",
+			err:     errors.New("exit code 1 - Could not detect protocol for plan 'something-odd'"),
+		},
+		{
+			name:    "unknown git plan protocol",
```

Comment:
Where is the git part of the test?

Reply:
Yeah the test case is actually more of `something unknown`. Good catch.

```suggestion
			name:    "unknown plan reference protocol",
```